spec/policies/accounting/transaction_policy_spec.rb

```diff
@@ -11,83 +11,91 @@
   let(:described_transaction) { Accounting::Transaction.new(project: loan) }
   subject(:policy) { described_class.new(user, described_transaction) }
 
-  shared_examples_for 'returns no reasons even if issues other than user role' do
+  shared_examples_for "returns no reasons even if issues other than user role" do
     it { expect(policy.read_only_reasons).to be_empty }
 
-    context 'with other issues present' do
+    context "with other issues present" do
       let(:loan_trait) { :completed }
       it { expect(policy.read_only_reasons).to be_empty }
     end
   end
 
-  context 'with non-admin' do
-    let(:user) { create(:user) }
-    forbid_all
-    it_behaves_like 'returns no reasons even if issues other than user role'
+  context "with non-admin" do
+    context "user has access to this division" do
+      let(:user) { create_member(division) }
+      forbid_all_but_index
+      it_behaves_like "returns no reasons even if issues other than user role"
+    end
+
+    context "user does not have access to this division" do
+      let(:user) { create(:user) }
+      forbid_all
+      it_behaves_like "returns no reasons even if issues other than user role"
+    end
   end
 
-  context 'with admin of wrong division' do
+  context "with admin of wrong division" do
     let(:user) { create(:user, :admin, division: create(:division)) }
     forbid_all
-    it_behaves_like 'returns no reasons even if issues other than user role'
+    it_behaves_like "returns no reasons even if issues other than user role"
   end
 
-  context 'with admin of division but not parent_division' do
+  context "with admin of division but not parent_division" do
     let(:user) { create(:user, :admin, division: division) }
-    forbid_all
-    it_behaves_like 'returns no reasons even if issues other than user role'
+    forbid_all_but_index
```

Comment:
logic change here: an admin on a division should be able to see the loan index for loans on that division

Reply:
Thanks for pointing these out.